tests/slr_jest/blockEditor/hook-callbacks.spec.js

```diff
@@ -0,0 +1,508 @@
+import { dispatch, select } from '@wordpress/data';
+import { applyFilters } from '@wordpress/hooks';
+import SeatType from '@tec/tickets/seating/blockEditor/header/seat-type';
+import { storeName } from '@tec/tickets/seating/blockEditor/store';
+import {
+	filterTicketIsAsc,
+	filterHeaderDetails,
+	filterSeatedTicketsAvailabilityMappedProps,
+	filterSetBodyDetails,
+	setSeatTypeForTicket,
+} from '@tec/tickets/seating/blockEditor/hook-callbacks';
+
+jest.mock('@wordpress/data', () => ({
+	select: jest.fn(),
+	createReduxStore: jest.fn(),
+	register: jest.fn(),
+	dispatch: jest.fn(),
+}));
+
+describe('setSeatTypeForTicket', () => {
+	beforeEach(() => {
```

Comment:
You can do this only once, in a wrapping `describe` block for the whole file. E.g. `describe('hook-callbacks', ...`

Reply:
So nice to know that! Thank you!